terracumber_config/tf_files/SUSEManager-4.2-beta-validation.tf

```diff
@@ -0,0 +1,973 @@
+
+  mirror = "minima-mirror-qam2.mgr.prv.suse.net"
+  use_mirror_images = true
+
+  testsuite          = true
+
+  provider_settings = {
+    pool        = "default"
+    bridge      = "br1"
+  }
+}
+
+module "base_newsle_ubuntu" {
+  providers = {
+    libvirt = libvirt.giediprime
+  }
+
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+  name_prefix = "suma-qam-42-"
+  use_avahi   = false
+  domain      = "mgr.prv.suse.net"
+  images      = [ "sles15o", "sles15sp1o", "sles15sp2o", "ubuntu1604o", "ubuntu1804o", "ubuntu2004o" ]
+
+  mirror = "minima-mirror-qam.mgr.prv.suse.net"
+  use_mirror_images = true
+
+  testsuite          = true
+
+  provider_settings = {
+    pool        = "default"
+    bridge      = "br1"
+    additional_network = "192.168.41.0/24"
+  }
+}
+
+module "server" {
+  source             = "./modules/server"
+  base_configuration = module.base_core.configuration
+  product_version    = "4.2-beta"
+  name               = "srv"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:89"
+    memory             = 40960
+    vcpu               = 10
+    data_pool            = "default"
+  }
+
+  repository_disk_size = 1500
+
+  auto_accept                    = false
+  monitored                      = true
+  disable_firewall               = false
+  allow_postgres_connections     = false
+  skip_changelog_import          = false
+  browser_side_less              = false
+  create_first_user              = false
+  mgr_sync_autologin             = false
+  create_sample_channel          = false
+  create_sample_activation_key   = false
+  create_sample_bootstrap_script = false
+  publish_private_ssl_key        = false
+  use_os_released_updates        = true
+  disable_download_tokens        = false
+  ssh_key_path                   = "./salt/controller/id_rsa.pub"
+  from_email                     = "root@suse.de"
+
+  //server_additional_repos
+
+}
+
+module "proxy" {
+  source             = "./modules/proxy"
+  base_configuration = module.base_core.configuration
+  product_version    = "4.2-beta"
+  name               = "pxy"
+  provider_settings = {
+    mac                = "aa:b2:92:42:00:8a"
+    memory             = 4096
+  }
+  server_configuration = {
+    hostname = "suma-qam-42-srv.mgr.prv.suse.net"
+    username = "admin"
+    password = "admin"
+  }
+  auto_register             = false
+  auto_connect_to_master    = false
+  download_private_ssl_key  = false
+  auto_configure            = false
+  generate_bootstrap_script = false
+  publish_private_ssl_key   = false
+  use_os_released_updates   = true
+  ssh_key_path              = "./salt/controller/id_rsa.pub"
+
+  //proxy_additional_repos
+
+}
+
+module "sles12sp4-client" {
+  providers = {
+    libvirt = libvirt.caladan
+  }
+  source             = "./modules/client"
+  base_configuration = module.base_old_sle.configuration
+  product_version    = "4.1-released"
```

Comment:
4.2 ?

Reply:
For now, we will only have server and proxy with 4.2 (that should deploy Alpha 2).
We let the clients with the released version of 4.1, because sumaform still not prepared for it.
Unless it has been done during this PR :D